### PR TITLE
bssh: use manned.org in more info link

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/bshell.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > A GUI tool for browsing for SSH/VNC servers on the local network.
 > See also: `bssh` and `bvnc`.
-> More information: <https://linux.extremeoverclocking.com/man/1/bssh>.
+> More information: <https://manned.org/bshell.1>.
```
Comment (blueskyson) [inline pages/common/bshell.md:5]: You can omit `.1`.
```suggestion
> More information: <https://manned.org/bshell>.
```

File: pages/common/bssh.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 > A GUI tool for browsing for SSH/VNC servers on the local network.
 > See also: `bvnc` and `bshell`.
-> More information: <https://linux.extremeoverclocking.com/man/1/bssh>.
+> More information: <https://manned.org/bssh.1>.
```
Comment (blueskyson) [inline pages/common/bssh.md:5]:
```suggestion
> More information: <https://manned.org/bssh>.
```

File: pages/common/bvnc.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 > A GUI tool for browsing for SSH/VNC servers on the local network.
 > See also: `bssh` and `bshell`.
-> More information: <https://linux.extremeoverclocking.com/man/1/bssh>.
+> More information: <https://manned.org/bvnc.1>.
```
Comment (blueskyson) [inline pages/common/bvnc.md:5]:
```suggestion
> More information: <https://manned.org/bvnc>.
```

**Review by mfrw — APPROVED** [review]
LGTM post @blueskyson suggestions are taken.

Thanks @vitorhcl

**Review by kbdharun — APPROVED** [review]

**Pull request merged**